### DISCUSSION TITLE
Chia - Log - Debug : Cannot Connect to host

### DISCUSSION THREAD

**Discussion by theng880205** [Chia-Network/chia-blockchain#6658] [Q&A]
I got a question regarding my chia farming. Everything is look good and smooth but in log - Debug it show that: Cannot connect to host. Is that normal or it will effect my farming?!thanks !your help for this is highly appreciated.[image](https://user-images.githubusercontent.com/85663605/121469762-6a781f00-ca00-11eb-887c-6093fded0ee6.jpeg)

**Comment by loppefaaret** [comment]
it is normal to get these errors on the internet, as long as it isn't your own local services, it is nothing to worry about

**Reply by theng880205** [reply]
Ok... thank you buddy
